### PR TITLE
Cleaning the project-1

### LINKED ISSUE #10: I've noticed a drag in performace. I think the project need to be cleaned
- [x] Have read the guidelines

The app tends to be slow at some points.

### PR DESCRIPTION
**Pull request by Mohameddekow**
This fixes #10

### CODE REVIEW

**Comment by Ericgacoki** [comment]
@Mohameddekow, thank you for submitting this PR. I'll review it before merging.

**Review by Ericgacoki — APPROVED** [review]
The comments were meant to be deleted. Approved.

File: app/src/main/java/com/ericg/sudofiemed/view/Details.kt [hunk 1]
```diff
@@ -26,14 +26,18 @@ class Details : Fragment() {
         _detailsBinding =
             DataBindingUtil.inflate(inflater, R.layout.fragment_details, container, false)
 
+        fun share(){
+            // val intent = Intent(requireActivity(),)
+        }
+
```
Comment (Ericgacoki) [inline app/src/main/java/com/ericg/sudofiemed/view/Details.kt]: @Mohameddekow, please remove incomplete lines of code like the one highlighted above.

**Review by Ericgacoki — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/ericg/sudofiemed/view/Details.kt (hunk 1 above)
Comment (Mohameddekow) [reply app/src/main/java/com/ericg/sudofiemed/view/Details.kt]: - [x] Alright, let me check it out.

**Review by Ericgacoki — APPROVED** [review]
Approved

**Comment by Ericgacoki** [comment]
Great work @Mohameddekow.

**Comment by Mohameddekow** [comment]
Thanks @Ericgacoki.

**Pull request merged**